sdk/framework/openapi.go

```diff
@@ -727,61 +749,3 @@ func cleanResponse(resp *logical.Response) *cleanedResponse {
 		Headers:  resp.Headers,
 	}
 }
-
-// CreateOperationIDs generates unique operationIds for all paths/methods.
-// The transform will convert path/method into camelcase. e.g.:
-//
-// /sys/tools/random/{urlbytes} -> postSysToolsRandomUrlbytes
-//
-// In the unlikely case of a duplicate ids, a numeric suffix is added:
-//
-//	postSysToolsRandomUrlbytes_2
-//
-// An optional user-provided suffix ("context") may also be appended.
-func (d *OASDocument) CreateOperationIDs(context string) {
```

Comment:
would removing this break api?

Reply:
Good catch. I'm surprised this function was exported in the first place. A quick github search seems to show only forks using this method. This doesn't seem to be something one would use outside of vault repo, so I think it's still safe to remove.